src/QueryBuilder/QueryBuilder/helpers/selectOptions.test.js

```diff
@@ -475,3 +475,19 @@ describe('getFilteredOptions', () => {
     expect(res).toEqual([{ label: '項目 - 持股 - 報表' }]);
   });
 });
+
+describe('getOperatorType', () => {
+  test('should return the correct operator group for an operator', () => {
+    expect(getOperatorType(OPERATORS.EQUAL)).toBe('comparison');
+    expect(getOperatorType(OPERATORS.NOT_EQUAL)).toBe('comparison');
+    expect(getOperatorType(OPERATORS.CONTAINS)).toBe('like');
+    expect(getOperatorType(OPERATORS.STARTS_WITH)).toBe('like');
+    expect(getOperatorType(OPERATORS.EMPTY)).toBe('null');
+    expect(getOperatorType(OPERATORS.IN)).toBe('arrayComparison');
+    expect(getOperatorType(OPERATORS.NOT_IN)).toBe('arrayComparison');
```

Comment:
Now that you have these nice constants like `OPERATORS_GROUPS_NAME.ARRAY_COMPARISON` you might as well use them in your tests too.

Reply:
Okay, will update it also! Thanks for comments !